apps/web/pages/event-types/[type].tsx

```diff
@@ -1469,7 +1469,7 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
                                           />
                                           <select
                                             id=""
-                                            className="block w-full rounded-sm border-gray-300 py-2 pl-3 pr-10 text-base text-sm focus:outline-none"
+                                            className="block w-full rounded-sm border-gray-300 py-2 pl-3 pr-10 text-sm focus:outline-none"
```

Comment:
Why is this change needed?

Reply:
@zomars text-base and text-sm both have the same rules and it was giving me a warning and I didn't see any harm in removing it